pyogrio/tests/test_geopandas_io.py

```diff
@@ -2067,3 +2067,22 @@ def test_non_utf8_encoding_shapefile_sql(tmp_path, use_arrow):
     )
     assert actual.columns[0] == mandarin
     assert actual[mandarin].values[0] == mandarin
+
+
+def test_write_kml_file_coordinates(tmp_path):
```

Comment:
```suggestion
def test_write_kml_coordinate_order(tmp_path):
```

Reply:
updated as suggested (manually due to conflict)